client/src/components/common/Inputs/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './style.css';
+import PropTypes from 'prop-types';
+
+
+const Input = (props) => {
+  const {
+    className, placeholder, type, name,
+  } = props;
+  return (
+    <input
+      type={type}
+      name={name}
+      className={className}
+      placeholder={placeholder}
+    />
+  );
+};
+
+
+Input.propTypes = {
+  className: React.string.isRequired,
```

Comment:
React.string is undefined. It should be PropTypes.string like the other attributes.